Unix/configeditor/configeditor.cpp

```diff
@@ -423,13 +445,29 @@ int MI_MAIN_CALL main(int argc, const char** argv)
             }
             ss << commentByte << prop << "=" << _CreateStringFromList(valueString,valueList);
         }
+        // Support querying a value from a property list
+        else if (!opts.query.empty())
+        {
+        	std::list<std::string> valueList;
+        	
+        	_ParseValueList(valueList, value);
+        	if (_IsValueInList(valueList, opts.query) != valueList.end())
+        	{
+        		cout << prop << " has the value " << opts.query << std::endl;
+        		return 0;
+        	}
+        	return 1;
+        }
         else
         {
             // Um, what are we supposed to do?
             err(ZT("Logic error, file %s, line %d"), scs(__FILE__), __LINE__);
         }
-
-        cout << ss.str() << std::endl;
+        
+        if (opts.query.empty())
```

Comment:
I guess @vimish should use Notepad++, as it default use this format, looks like you should make some change on the setting, I didn't try it, I also use Notepad++ and I deleted the spaces in new lines manually...
for the indent in Notepad++, I have set in setting to make 4 spaces instead of tab.